### PR TITLE
Fix CSS lint errors

### PR DESCRIPTION
**Pull request by fernandojsg**
Also renamed `test:css` to `lint:css`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,12 +11,12 @@
     "dist:min": "NODE_ENV=production AFRAME_DIST=true npm run build",
     "ghpages": "npm run preghpages && node ./scripts/gh-pages",
     "lint": "npm run lintfile src/",
+    "lint:css": "stylelint src/css/main.css"
     "lintfile": "node --harmony node_modules/.bin/eslint",
     "preghpages": "npm run dist:min && shx rm -rf gh-pages && shx mkdir gh-pages && shx cp -r assets dist example index.html gh-pages && mv gh-pages/dist gh-pages/build && shx sed -i http://localhost:3333 .. gh-pages/example/index.html && shx sed -i ../dist/aframe-inspector.js dist/aframe-inspector.min.js gh-pages/example/index.html",
     "start": "cross-env NODE_ENV=dev webpack-dev-server --progress --colors --hot -d --open --host 0.0.0.0",
     "test": "jest --watch",
     "test:ci": "jest",
```
Comment (ngokevin) [inline package.json]: trailing comma?

File: package.json [hunk 2]
```diff
@@ -11,12 +11,12 @@
     "dist:min": "NODE_ENV=production AFRAME_DIST=true npm run build",
     "ghpages": "npm run preghpages && node ./scripts/gh-pages",
     "lint": "npm run lintfile src/",
+    "lint:css": "stylelint src/css/main.css"
```
Comment (ngokevin) [inline package.json]: comma

File: package.json (hunk 1 above)
Comment (fernandojsg) [reply package.json]: fixed thanks!

File: package.json (hunk 2 above)
Comment (fernandojsg) [reply package.json]: :+1:

**Pull request merged**